commands.c

```diff
@@ -236,7 +238,7 @@ bool cg_navigate_marked(arg_t n)
 {
 	int d, i;
 	int new = fileidx;
-	
+
```

Comment:
I think we've taken care of this in the "consistency" PR. Best to revert it here and keep the patch to functional changes only.